buildSrc/src/main/groovy/nva.identity.service.java-common-conventions.gradle

```diff
@@ -0,0 +1,250 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    // Apply the java Plugin to add support for Java.
+    id 'java'
+
+    id 'jacoco'
+    id 'checkstyle'
+    id 'pmd'
+    //id "nebula.lint"
+    id 'maven-publish'
+    id 'base'
+}
+
+//gradleLint {
+//  rules  = ['all-dependency']
+//}
+
+repositories {
+    // Use Maven Central for resolving dependencies.
+    mavenCentral()
+
+    maven { url 'https://jitpack.io' }
+    maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
+}
+
+project.ext {
+    nvaCommonsVersion = '1.12.0'
+    awsSdkVersion ="1.12.59"
+    jacksonVersion ="2.10.3"
+    xraySdkVersion = '2.9.1'
+    DynamoDBLocalVersion = '1.15.0'
+    nvaTestUtilsVersion = '0.1.13'
+    zalandoProblemVersion = '0.25.0'
+    awsLambdaCoreVersion = '1.2.1'
+    jupiterVersion = "5.6.0"
+}
+
+dependencies {
+    constraints {
+        implementation group: 'com.github.bibsysdev', name: 'core', version: project.ext.nvaCommonsVersion
```

Comment:
I am not sure that dependecies for all projects should be merged here.

Reply:
They are only version constraints